src/wp-admin/includes/media.php

```diff
@@ -1336,23 +1415,35 @@ function get_attachment_fields_to_edit($post, $errors = null) {
 function get_media_items( $post_id, $errors ) {
 	$attachments = array();
 	if ( $post_id ) {
-		$post = get_post($post_id);
-		if ( $post && $post->post_type == 'attachment' )
-			$attachments = array($post->ID => $post);
-		else
-			$attachments = get_children( array( 'post_parent' => $post_id, 'post_type' => 'attachment', 'orderby' => 'menu_order ASC, ID', 'order' => 'DESC') );
+		$post = get_post( $post_id );
+		if ( $post && $post->post_type == 'attachment' ) {
+			$attachments = array( $post->ID => $post );
+		} else {
+			$attachments = get_children(
+				array(
+					'post_parent' => $post_id,
+					'post_type'   => 'attachment',
+					'orderby'     => 'menu_order ASC, ID',
+					'order'       => 'DESC',
+				)
+			);
+		}
 	} else {
-		if ( is_array($GLOBALS['wp_the_query']->posts) )
-			foreach ( $GLOBALS['wp_the_query']->posts as $attachment )
-				$attachments[$attachment->ID] = $attachment;
+		if ( is_array( $GLOBALS['wp_the_query']->posts ) ) {
+			foreach ( $GLOBALS['wp_the_query']->posts as $attachment ) {
+				$attachments[ $attachment->ID ] = $attachment;
+			}
+		}
 	}
 
 	$output = '';
 	foreach ( (array) $attachments as $id => $attachment ) {
-		if ( $attachment->post_status == 'trash' )
+		if ( $attachment->post_status == 'trash' ) {
 			continue;
-		if ( $item = get_media_item( $id, array( 'errors' => isset($errors[$id]) ? $errors[$id] : null) ) )
+		}
+		if ( $item = get_media_item( $id, array( 'errors' => isset( $errors[ $id ] ) ? $errors[ $id ] : null ) ) ) {
```

Comment:
assignment in if statement

Reply:
As per review meeting, this is fine to keep the change